Packs/Base/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -0,0 +1,585 @@
+                                     else VERDICT_ERROR_COLOR % x.get('verdict') for x in results]
+    df_summary_json = df_summary.to_dict(orient='records')
+    return_entry = {
+        "Type": entryTypes["note"],
+        "ContentsFormat": formats['json'],
+        "HumanReadable": tableToMarkdown("Phishing prediction summary for URLs", df_summary_json,
+                                         headers=['Url', KEY_FINAL_VERDICT]),
+        "Contents": df_summary_json,
+        "EntryContext": {'DBotPredictURLPhishing': df_summary_json}
+    }
+    if tag is not None:
+        return_entry["Tags"] = ['SimilarIncidents_{}'.format(tag)]
+    demisto.results(return_entry)
+    return df_summary_json
+
+
+def return_detailed_summary(results, number_entries_to_return):
+    outputs = []
+    severity_list = [x.get('score') for x in results]
+    indice_descending_severity = np.argsort(-np.array(severity_list), kind='mergesort')
+    for i in range(min(number_entries_to_return, len(results))):
+        index = indice_descending_severity[i]
+        pred_json = results[index].get('pred_json')
+        url = results[index].get('url')
+        is_white_listed = results[index].get('is_white_listed')
+        output_rasterize = results[index].get('output_rasterize')
+        summary_json = return_entry_summary(pred_json, url, is_white_listed, output_rasterize)
+        outputs.append(summary_json)
+        outputs = [x for x in outputs if x]
+    return outputs
+
+
+def save_model_in_demisto(model):
+    encoded_model = base64.b64encode(dill.dumps(model))  # guardrails-disable-line
+    res = demisto.executeCommand('createMLModel', {'modelData': encoded_model.decode('utf-8'),
+                                                   'modelName': URL_PHISHING_MODEL_NAME,
+                                                   'modelLabels': [MALICIOUS_VERDICT, BENIGN_VERDICT],
+                                                   'modelOverride': 'true',
+                                                   'modelHidden': True,
+                                                   'modelType': 'url_phishing',
+                                                   'modelExtraInfo': {
+                                                       OOB_MAJOR_VERSION_INFO_KEY: model.major,
+                                                       OOB_MINOR_VERSION_INFO_KEY: model.minor}})
+    if is_error(res):
+        return_error(get_error(res))
+
+
+def load_demisto_model():
+    model_64_str = get_model_data(URL_PHISHING_MODEL_NAME)[0]
+    model = decode_model_data(model_64_str)
+    return model
+
+
+def main():
+    msg_list = []
+    exist, demisto_major_version, demisto_minor_version = oob_model_exists_and_updated()
+    reset_model = demisto.args().get('resetModel', 'False') == 'True'
+    if exist:
+        demisto.results("Model version in demisto: %s.%s" % (demisto_major_version, demisto_minor_version))
```

Comment:
I'm not sure we need this as result always, maybe add "debug" argument and only when it set to True return it